src/SFA.DAS.FindEpao.Web/Views/Courses/CourseEpao.cshtml

```diff
@@ -122,24 +137,23 @@
                     </div>
                 }
                 
+                <br /><br />
+
                 <div class="govuk-summary-list__row">
-                    <dt class="govuk-summary-list__key">Apprenticeship training courses</dt>
-                    <dd class="govuk-summary-list__value">
-                    @if (Model.AllCourses.Count == 1)
-                    {
-                        @Model.AllCourses[0].Title <span class="das-no-wrap">(level @Model.AllCourses[0].Level)</span>
-                    }
-                    else
-                    {
-                        <ul class="govuk-list govuk-list--bullet">
-                        @foreach (var course in Model.AllCourses)
-                        {
-                            <li>@course.Title <span class="das-no-wrap">(level @course.Level)</span></li>
-                        }
-                        </ul>
-                    }
-                    </dd>
-                </div>
+                    
+                    <p class="govuk-!-font-weight-bold das-no-wrap">Apprenticeship training courses<br />
+
+                        <table class="govuk-body das-no-wrap">
```

Comment:
Again, the CSS classes and markup look wrong to me. Is it a table or a summary list? If its a table, it needs table headings